cluster/manifests/skipper/deployment.yaml

```diff
@@ -128,6 +128,41 @@ spec:
           - name: filters
             mountPath: /etc/config/default-filters
 {{ end }}
+{{ if eq .ConfigItems.tokeninfo_url "http://127.0.0.1:9021/oauth2/tokeninfo" }}
```

Comment:
nit: I think it would be simpler if we just had a config item: `enable_local_tokeninfo` otherwise you have to remember that correct value of `http://127.0.0.1:9021/oauth2/tokeninfo` to configure it.

Reply:
But this will need to create a lot more configuration overhead, because you would need to have it in multiple places.
For me this was more DRY, so I disagree with being a nit.